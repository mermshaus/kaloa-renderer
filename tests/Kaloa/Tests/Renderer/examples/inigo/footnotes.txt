Test[fn] Toast[fn]

[fnt]Hello world![fn]

How are things?[/fnt]

This is not about love.

[fnt]Tasty![/fnt]

[fnt]foo[/fnt]