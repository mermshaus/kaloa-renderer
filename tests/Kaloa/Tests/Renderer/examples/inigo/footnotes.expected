<p>Test[1] Toast[2]</p>
<p>This is not about love.</p>
<ol>
<li><p>Hello world![3]</p>

<p>How are things?</p>
</li>
<li>Tasty!</li>
<li>foo</li>
</ol>